duckduckgo/iOS
# Minor Autofill UI updates

Task/Issue URL: https://app.asana.com/0/0/1205435116192702/f, https://app.asana.com/0/0/1205495150811753/f, https://app.asana.com/0/0/1205495150811762/f, https://app.asana.com/0/0/1205533338187716/f
Tech Design URL:
CC:

**Description**:
Copy and layout updates resulting from "on by default" PFR



**Steps to test this PR**:

Note: Autofill is intentionally (and temporarily) enabled by default for the purposes of the Ship Review and I'll be reverting before merging, which will also fix this failing unit test `AppUserDefaultsTests.testDefaultAutofillStateIsFalse()`

1. In both empty state and when logins are present, confirm the label "Logins are stored securely on your device" is always visible as the section footer and screen looks similar to the figma design (note there was one [change](https://app.asana.com/0/0/1205541917560712/1205431733474528/f) requested from the design review -> the label is now indented) :
![image](https://github.com/duckduckgo/iOS/assets/91189922/cfff9eed-b936-4a1d-ad55-18834d33d7da)
2. In the Autofill prompts, confirm the max width of the content is now restricted to 480px especially in iPhone landscape and on iPad as per figma:
![image](https://github.com/duckduckgo/iOS/assets/91189922/677f48a4-c122-45bd-93f4-df96369c8058)
3. In the Save Logins prompt, confirm the body copy is now `Logins are stored securely on your device in the Logins menu`
4. In the Password Generation prompt, confirm the title is now `Use a strong password from DuckDuckGo?`, the body is now `Passwords are stored securely on your device in the Logins menu.` and the CTA copy is `Use Strong Password`
5. Confirm the prompt to disable Autofill now triggers after 2 instead of 3 dismisses of the Save Login prompt (across 2 different sites)



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
